### ISSUE TITLE
More Info Screen - VoiceOver Crashes "Methodology" Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#75]
When I go to the "Methodology" screen in the "More Info" section using VoiceOver (screen reader) the application crashes.

**Comment by carolynhuang** [comment]
Is this an ipad issue? Developers were unable to replicate.

**Comment by jfc3-dol** [comment]
It's no longer crashing the application when reading this screen with VoiceOver.

Closing issue.
